influxdb/server.sls

```diff
@@ -59,7 +80,7 @@ influxdb_service:
   {% set admin_url = "http://{}:{}/query?u={}&p={}".format(server.http.bind.address, server.http.bind.port, server.admin.user.name, server.admin.user.password) %}
 influxdb_create_admin:
   cmd.run:
-  - name: curl -f -S -POST "{{ url_for_query }}" {{ query_create_admin }} || curl -f -S -POST "{{ admin_url }}" {{ query_create_admin }}
+  - name: curl -f -S -GET "{{ url_for_query }}" {{ query_create_admin }} || curl -f -S -GET "{{ admin_url }}" {{ query_create_admin }}
```

Comment:
I do not understand why this POST->GET change is required because:
1) it does not match what Influx API documentation is relating
2) it goes against standard REST specifications anyways